pkg/constants/aggs.go

```diff
@@ -0,0 +1,19 @@
+package constants
+
+type AggregationConfig struct {
+	Field string
+	Type  string
+}
+
+var AggregationConfigs = map[string]map[string]AggregationConfig{
+	"BookStats": {
+		"distinct_authors": {
```

Comment:
This agg name definitely should have a shared const with its other usage in `pkg/query/books.go`